posthog/api/decide.py

```diff
@@ -517,6 +517,24 @@ def _session_recording_domain_not_allowed(team: Team, request: HttpRequest) -> b
     return team.recording_domains and not on_permitted_recording_domain(team.recording_domains, request)
 
 
+# KLUDGE: this is duplicated in posthog/models/remote_config.py
```

Comment:
rather than collapse this duplication that confused me, i've at least marked it for the future traveller

Reply:
One more word to google.

But if it's exactly the same code - why not to move it to some sort of utils? Again, I assume there's a specific reason for that, I just want to know it :)

Reply:
only laziness :)

Reply:
oh, makes sense then. would still be a bit happier to have it reused instead of the comment (as it suggests the change later anyway) 😄